src/extent.c

```diff
@@ -2124,7 +2124,14 @@ extent_util_stats_verbose_get(tsdn_t *tsdn, const void *ptr,
 	} else {
 		*bin_nfree = *bin_nregs = 0;
 	}
-	*slabcur_addr = extent_addr_get(bin->slabcur);
-	assert(*slabcur_addr != NULL);
+	if (bin->slabcur != NULL) {
+		*slabcur_addr = extent_addr_get(bin->slabcur);
+		assert(*slabcur_addr != NULL);
+	} else {
+		extent_t *first_nonfull_extent =
+		    extent_heap_first(&bin->slabs_nonfull);
+		*slabcur_addr = first_nonfull_extent == NULL ? NULL :
+		    extent_addr_get(first_nonfull_extent);
+	}
```

Comment:
maybe refactor a bit --

    if (bin->slabcur != NULL) {
		slab = extent_addr_get(bin->slabcur);
    } else {
		slab = extent_heap_first(&bin->slabs_nonfull);
    }

    *slabcur_addr = slab != NULL ? extent_addr_get(first_nonfull_extent) : NULL;
    
Did we document this API yet? If so we need to update the manual as well.

Reply:
It's an experimental API. The only documentation was a block of comment in `ctl.c`. Updated it.